lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -917,6 +915,60 @@ class ObjCPrinter : private DeclVisitor<ObjCPrinter>,
     }
     return hasPrintedAnything;
   }
+    
+  void printRenameForDecl(const AvailableAttr *AvAttr, const Decl *D,
+                          bool includeQuotes) {
+    if (AvAttr->Rename.empty() && isa<ValueDecl>(D))
+      return ;
+    
+    auto renamedParsedDeclName = parseDeclName(AvAttr->Rename);
+    auto renamedDeclName = renamedParsedDeclName.formDeclName(D->getASTContext());
+    
+    auto declContext = D->getDeclContext();
+    ValueDecl *renamedFuncDecl = nullptr;
```

Comment:
Still not necessarily a function anymore. :-) Also, may as well mark this `const`.

Reply:
`Still not necessarily a function anymore` I'm not sure what you mean here

Reply:
`may as well mark this const` but I need to change the renamedFuncDecl pointer after this. Or I misunderstood your point?